src/main/resources/screens/PSU_TRAIGE_COMPLAINT.json

```diff
@@ -0,0 +1,35 @@
+{
+  "props": {},
+  "title": "Triage complaint",
+  "defaultActionLabel": "Submit",
+  "fields": [
+    {
+      "validation": [
+        "required"
+      ],
+      "props": {
+        "choices": [
+          {
+            "label": "Yes",
+            "value": "Accept"
+          },
+          {
+            "label": "No - send back to IE Detention",
+            "value": "ReturnCase"
+          },
+          {
+            "label": "No - send to team not to DECS",
+            "value": "CloseCase"
+          }
+        ],
+        "labelClassName": "govuk-label--m"
+      },
+      "component": "radio",
+      "name": "CompliantType",
```

Comment:
Should this be called complaint type? It doesn't really refer to a complaint type, I think something like PsuTriageOutcome would be more appropriate

Reply:
Review comments incorporated.